linera-sdk/src/lib.rs

```diff
@@ -6,29 +6,24 @@
 //! A Linera application consists of two WebAssembly binaries: a contract and a service.
 //! In both binaries, there should be a shared application state. The state is a type that
 //! represents what the application would like to persist in storage across blocks, and
-//! must implement the [`Contract`](crate::Contract) trait in the contract binary and the
-//! [`Service`](crate::Service) trait in the service binary.
-//!
-//! The application can select between two storage backends to use. Selecting the storage
-//! backend is done by specifying both the [`Contract::Storage`](crate::Contract::Storage)
-//! and the [`Service::Storage`](crate::Service::Storage) associated types.
+//! must implement [`State`](crate::State) trait in order to specify how the state should be loaded
+//! from and stored to the persistent key-value storage. An alternative is to use the
+//! [`linera-views`](https://docs.rs/linera-views/latest/linera_views/index.html), a framework that
+//! allows loading selected parts of the state. This is useful if the application's state is large
+//! and doesn't need to be loaded in its entirety for every execution. By deriving
+//! [`RootView`](crate::views::RootView) on the state type it automatically implements the
+//! [`State`] trait.
 //!
-//! The [`SimpleStateStorage`](crate::SimpleStateStorage) backend stores the application's
-//! state type by serializing it into binary blob. This allows the entire contents of the
-//! state to be persisted and made available to the application when it is executed.
-//!
-//! The [`ViewStateStorage`](crate::ViewStateStorage) backend stores the application's
-//! state using the
-//! [`linera-views`](https://docs.rs/linera-views/latest/linera_views/index.html), a
-//! framework that allows loading selected parts of the state. This is useful if the
-//! application's state is large and doesn't need to be loaded in its entirety for every
-//! execution.
+//! The [`Contract`](crate::Contract) trait in the contract binary and the
+//! [`Service`](crate::Service) trait in the service binary.
```

Comment:
I don't understand; do you mean "is used in"?

Reply:
Oops! That should have been removed, thanks!